### PR TITLE
about.html: use vs. not vs for versus

### PR DESCRIPTION
**Pull request by alanpost**
The other use on this page uses vs. h/t geoffwozniak.

### CODE REVIEW

File: app/views/home/about.html.erb [hunk 1]
```diff
@@ -143,7 +143,7 @@
   </div>
 
   <p>
-  Less than 1% of stories or comments get flagged, and users are not automatically punished by flags.
+  Fewer than 1% of stories or comments get flagged, and users are not automatically punished by flags.
```
Comment (alanpost) [inline app/views/home/about.html.erb:146]: I draw a reviewer's attention to la wik's discussion of this topic:

https://en.wikipedia.org/wiki/Fewer_versus_less

Comment (pushcx) [reply app/views/home/about.html.erb:146]: Yeah, pass on this one, it doesn't read right.

Comment (alanpost) [reply app/views/home/about.html.erb]: Undone.

File: app/views/home/about.html.erb [hunk 2]
```diff
@@ -74,7 +74,7 @@
   </p>
 
   <p>
-  Some things that are off-topic here but popular on larger, similar sites: entrepreneurship, management, news about companies that employ a lot of programmers, investing, world events, anthropology, and defining the single morally correct economic and political system for the entire world when we can't even settle tabs vs. spaces.
+  Some things that are off-topic here, but popular on larger, similar sites: entrepreneurship, management, news about companies that employ a lot of programmers, investing, world events, anthropology, and defining the single morally correct economic and political system for the entire world when we can't even settle tabs vs. spaces.
```
Comment (pushcx) [inline app/views/home/about.html.erb:77]: I can see why one might set off this clause, but I prefer not to break up this sentence.

Comment (alanpost) [reply app/views/home/about.html.erb:77]: It's undone.

**Pull request merged**